tests/wpt/metadata/html/semantics/scripting-1/the-script-element/module/dynamic-import/no-active-script-manual-module.html.ini

```diff
@@ -0,0 +1,2 @@
+[no-active-script-manual-module.html]
+  expected: TIMEOUT
```

Comment:
These 2 tests are introduced in web-platform-tests/wpt#14157 for testing "no active script" triggered by user click; however, there's no way to trigger an user click in WPT for now (see web-platform-tests/wpt#14158) so they were expected TIMEOUT.

I was disappointed that we always need to wait for a few seconds for these 2 expected TIMEOUT tests but, luckily, Domenic changed filename for these 2 files to match the manual test naming convention in web-platform-tests/wpt#24565 so they will be seen as manual tests and won't be run in CI.

I also checked them locally and yes, both of them are PASS!

(I will remove both of this expected TIMEOUT when next WPT sync are merged!)